src/Symfony/Component/Workflow/Marking.php

```diff
@@ -12,41 +12,45 @@
 namespace Symfony\Component\Workflow;
 
 /**
- * Marking contains the place of every tokens.
+ * A base Marking which contains the state of the
+ * state of a Workflow or a StateMachine a representation
+ * by places with tokens.
  *
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ * @author Jules Pietri <jules@heahprod.com>
  */
-class Marking
+abstract class Marking
```

Comment:
Not sure this class is really needed.. why not a pointer interface and 2 final implementations instead?